docs/pages/database-access/guides/rds.mdx

```diff
@@ -18,99 +18,347 @@ This guide will help you to:
+Run the command below on your Linux host. Assign <Var
+name="teleport.example.com:443" /> to the host and port of your Teleport Proxy
+Service or Enterprise Cloud/Team tenant. Assign <Var name="us-west-1" /> to the
+region associated with the RDS databases you would like Teleport to discover.
 
 ```code
 $ teleport db configure create \
    -o file \
-   --proxy=teleport.example.com:3080 \
+   --proxy=<Var name="teleport.example.com:443" />  \
    --token=/tmp/token \
-   --rds-discovery=us-west-1
+   --rds-discovery=<Var name="us-west-1" />
 ```
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
+The command will generate a Database Service configuration with RDS/Aurora
+database auto-discovery enabled on the configured region and place it at the
+`/etc/teleport.yaml` location.
 
-```code
-$ teleport db configure create \
-   -o file \
-   --proxy=mytenant.teleport.sh:443 \
-   --token=/tmp/token \
-   --rds-discovery=us-west-1
-```
 
-</ScopedBlock>
+</TabItem>
+<TabItem label="Kubernetes Cluster">
 
-The command will generate a Database Service configuration with RDS/Aurora
-database auto-discovery enabled on the `us-west-1` region and place it at the
-`/etc/teleport.yaml` location.
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+</TabItem>
+</Tabs>
 
-## Step 3/6. Create an IAM policy for Teleport
+## Step 3/6. Create IAM policies for Teleport
 
-Teleport needs AWS IAM permissions to be able to:
+The Teleport Database Service needs AWS IAM permissions to be able to:
 
 - Discover and register RDS instances and Aurora clusters.
 - Configure IAM authentication for them.
 
-Before you can generate IAM permissions, you must provide the Teleport Database
-Service access to AWS credentials.
+In this step, we will show you how to provide the Teleport Database Service
+access to AWS credentials by using the Teleport configuration file we created
+earlier to automatically create and attach IAM policies:
+
+<Tabs>
+<TabItem label="Linux Host">
+
+Follow these instructions on your Linux host.
 
 (!docs/pages/includes/aws-credentials.mdx service="the Database Service"!)
 
 (!docs/pages/includes/database-access/aws-bootstrap.mdx!)
 
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+
+Follow these instructions on your local workstation.
+
+Create an IAM policy document that allows an IAM identity to connect to your RDS
+database. Assign <Var name="rds-region" /> to the name of the AWS region where
+your RDS database is running, <Var name="aws-account" /> to your AWS account
+number, and <Var name="resource-id" /> to the resource ID of your RDS or Aurora
+database:
+
+```code
+$ cat > connect.json << EOF
+{
+   "Version": "2012-10-17",
+   "Statement": [
+      {
+         "Effect": "Allow",
+         "Action": [
+             "rds-db:connect"
+         ],
+         "Resource": [
+             "arn:aws:rds-db:<Var name="rds-region" />:<Var name="aws-account" />:dbuser:<Var name="resource-id" />/*"
```

Comment:
Instead of variable names in the content I would have these as env variables assigned to the variable names. I'm finding some of these outputs are malformed.

Reply:
If the outputs are malformed, I would treat that as a bug with the `Var` component, which we can fix. What issues are you seeing?

Reply:
I'll post an example after I recreate

Reply:
Thanks! We fixed an issue with copying the `Var` component's value recently, so you might need to rebase your local `docs` clone against `main`.

I've also noticed that the trust policy document I included in the draft has some errors in it, so I'll work on fixing that and see if the fixes resolve my issues connecting to my RDS database.

Reply:
Alternatively you could use the `var` syntax and just provide the file contents without the `cat` command, giving a little more wiggle room for folks with their own preferred methods of creating/editing files.